pkg/controller/remoteingress/remoteingress_controller.go

```diff
@@ -214,32 +214,32 @@ func rawExtensionsFromClusterDeployment(rContext *reconcileContext) []runtime.Ra
 	return rawList
 }
 
-// secretRefrenceFromClusterDeployment will return the slice of hivev1.SecretReference objects
-// (really the syncSet.Spec.SecretReferences) to satisfy the ingress config for the clusterDeployment
-func secretRefrenceFromClusterDeployment(rContext *reconcileContext) []hivev1.SecretReference {
-	secretReferences := []hivev1.SecretReference{}
+// secretMappingsFromClusterDeployment will return the slice of hivev1.SecretMapping objects
+// (really the syncSet.Spec.SecretMappings) to satisfy the ingress config for the clusterDeployment
+func secretMappingsFromClusterDeployment(rContext *reconcileContext) []hivev1.SecretMapping {
+	secretMappings := []hivev1.SecretMapping{}
 
 	for _, cbSecret := range rContext.certBundleSecrets {
-		secretReference := hivev1.SecretReference{
-			Source: corev1.ObjectReference{
+		secretMapping := hivev1.SecretMapping{
+			Source: hivev1.SecretReference{
 				Namespace: cbSecret.Namespace,
 				Name:      cbSecret.Name,
 			},
-			Target: corev1.ObjectReference{
+			Target: hivev1.SecretReference{
 				Namespace: remoteIngressConrollerSecretsNamespace,
```

Comment:
Note, for myself so that I can find it later, that there is a typo here where Controller is incorrectly spelled Conroller.